### PR TITLE
doc : add a section on diagnosing TPM issues

### PR DESCRIPTION
**Pull request by shjala**

### CODE REVIEW

**Review by OhmSpectator — APPROVED** [review]
Looks good

File: docs/DEBUGGING.md [hunk 1]
```diff
@@ -300,3 +300,65 @@ The uuid of the device running eve-os can be obtained by running following comma
...
+In addition to error logs, EVE saves a snapshot of TPM's Stored Measurement Log (SML) on two occasions, 1) when vault key is successfully sealed and 2) when the unseal operation fails. This information can be helpful to pin point the issue by parsing the logs and checking the TPM events. The log files can be collected from `/persist/status`:
+
+```text
+# ls /persist/status/tpm_measurement*
+/persist/status/tpm_measurement_seal_success-tpm0
+/persist/status/tpm_measurement_seal_success-tpm0-backup
+/persist/status/tpm_measurement_unseal_fail-tpm0-backup
+```
+
+Further parsing and diffing the files is possible using tpm2-tools:
```
[the remark this answers is not in the document]
Comment (shjala) [reply docs/DEBUGGING.md:334]: the whole tpm2-tools suite is in the VTPM container, but for some reason that I'm going to debug `tpm2_eventlog` command is not working properly. Once I have fixed that I will change the do on how to use it from VTPM container.

[the remark this answers is not in the document]
Comment (shjala) [reply docs/DEBUGGING.md:334]: Weirdly I can run  `tpm2_eventlog` from debug container, but not vtpm container: 
```
linuxkit-525400123456:/# eve enter vtpm
linuxkit-525400123456:/# tpm2 eventlog /home/evtlog.bin
ERROR: Unable to run eventlog
linuxkit-525400123456:/# exit
linuxkit-525400123456:/# eve enter debug
linuxkit-525400123456:/# chroot /containers/services/vtpm/rootfs tpm2 eventlog /home/evt.bin | head
---
version: 1
events:
- EventNum: 0
  PCRIndex: 0
  EventType: EV_NO_ACTION
  Digest: "0000000000000000000000000000000000000000"
  EventSize: 45
  SpecID:
  - Signature: Spec ID Event03
linuxkit-525400123456:/#
```

I guess it is due to a memory limitation for vtpm container.

[the remark this answers is not in the document]
Comment (eriknordmark) [reply docs/DEBUGGING.md:334]: > Weirdly I can run `tpm2_eventlog` from debug container, but not vtpm container:

Isn't this due to the apparmor config for the vtpm container?

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**